src/wazuh_modules/vulnerability_scanner/tests/mocks/MockCleanUpData.hpp

```diff
@@ -15,6 +15,8 @@
 #include "cleanUpData.hpp"
 #include "gmock/gmock.h"
 
+constexpr int CLEAN_UP_DATA_ID {6};
```

Comment:
Improve (design): If we change the HANDLERS enum, this will break

```suggestion
constexpr int CLEAN_UP_DATA_ID {HANDLERS::CLEAN_UP_DATA};
```

Reply:
This was the flaky implementation I discussed before about this mocks. I'll try to reimplement them then.

Reply:
I'll create this issue to address this flaky implementation later https://github.com/wazuh/wazuh/issues/22139